dbt-cta/action_builder/models/sources.yml

```diff
@@ -3,137 +3,365 @@ sources:
+          - not_null
   - name: activity_events_base
-  - name: addresses_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: addresses_electoral_districts_base
+  - name: addresses_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: api_keys_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: ar_internal_metadata_base
+    columns:
+      - name: key
+        tests:
+          - unique
+          - not_null
   - name: assessments_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: attachments_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: campaign_configuration_tag_categories_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: campaign_context_notes_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: campaign_entity_type_configurations_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: campaign_entity_types_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: campaign_participations_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: campaign_topline_setting_items_base
-  - name: campaigns_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: campaigns_entities_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: campaigns_entity_connection_types_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: campaigns_tag_categories_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: campaigns_tag_groups_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: campaigns_tags_base
-  - name: contact_attempts_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
+  - name: campaigns_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: contact_sources_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: contact_statuses_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: electoral_districts_base
+    columns:
+      - name: ocd_id
+        tests:
+          - unique
+          - not_null
   - name: emails_base
+    columns:
+      - name: id
+        tests:
+          - unique
+          - not_null
   - name: entities_base
-  - name: entity_connection_types_base
```

Comment:
It looks like this table is getting removed completely, is that expected?

Reply:
oh, yeah so an annoying property of the Action Builder sync is that for one partner there are a few tables that we cannot sync because of data replication conflicts with the database. But since the syncs are all sharing the same dbt project, the dbt yaml has to work for all of them. And if we include references to tables that don't exist, the dbt will fail. We could create empty versions of those tables on an as-needed basis, though, and that might be the more reasonable way to handle this... I shall contemplate

Reply:
COOL FACT, since the DAG runs all the table models individually, it actually doesn't seem to mind that sources.yml includes sources that might not be present for a particular partner!

Reply:
ooh that is a cool fact!